protocol/message_persistence.go

```diff
@@ -1019,10 +1038,20 @@ func (db sqlitePersistence) PinnedMessageByChatIDs(chatIDs []string, currCursor
        ON
        dm.author_id = dm_author.id
 
-      LEFT JOIN 
+       LEFT JOIN 
               discord_message_attachments dm_attachment
-      ON        
-      dm.id = dm_attachment.discord_message_id
+			 ON        
+       dm.id = dm_attachment.discord_message_id
+
+			 LEFT JOIN 
+							discord_messages m2_dm
+			 ON        
+			 m2.discord_message_id = m2_dm.id
+
+				LEFT JOIN 
+							discord_message_authors m2_dm_author
+			 ON        
+			 m2_dm.author_id = m2_dm_author.id
```

Comment:
The indentations look crazy on Github. Not sure if spaces and tabs might have been mixed?

Reply:
I know, unfortunately they aren't honored by the lint-fix tool. I see if I can get them aligned.